test/quantization/algorithm/test_smooth_quant.py

```diff
@@ -75,3 +79,178 @@ def test_value(self):
             atol=1e-5,
             err_msg=f"Value mismatches.\nbefore result: {base_output}\nafter result: {target_output}",
         )
+
+
+# ────────────────────────────────────────────────────────────
+# Faux fairseq injection (so the handler path works w/o fairseq)
```

Comment:
Just curiosity. Is the term "faux" is used somewhere? I've seen "mock" or "fake" more frequently.

Reply:
A term is suggested by a generative model :smile:.